packages/@uppy/aws-s3-multipart/src/index.test.ts

```diff
@@ -257,16 +258,16 @@ describe('AwsS3Multipart', () => {
         .mock.calls[0][1]
 
       expect(completeCall.parts).toEqual([
-        { ETag: 'test', PartNumber: 1 },
-        { ETag: 'test', PartNumber: 2 },
-        { ETag: 'test', PartNumber: 3 },
-        { ETag: 'test', PartNumber: 4 },
-        { ETag: 'test', PartNumber: 5 },
-        { ETag: 'test', PartNumber: 6 },
-        { ETag: 'test', PartNumber: 7 },
-        { ETag: 'test', PartNumber: 8 },
-        { ETag: 'test', PartNumber: 9 },
-        { ETag: 'test', PartNumber: 10 },
+        { ETag: 'test', etag: 'test', PartNumber: 1 },
```

Comment:
This looks a bit weird but I think it eventually makes sense to have everything lowercase but for now maintain backwards compatibility